src/tests/users.test.ts

```diff
@@ -0,0 +1,203 @@
+import { expect, test } from '@/fixtures/fixtures';
+import ImpersonateBanner from '@/pages/ImpersonateBanner';
+import Navbar from '@/pages/Navbar';
+import { UserType } from '@/types';
+import AppConfig from '@/utils/AppConfig';
+
+const formData: UserType = {
+  username: 'test1',
+  firstName: 'test1',
+  lastName: 'test1',
+  password: 'test1',
+};
+
+test.beforeEach(
+  async ({ page, navbar, userListPage, createUserPage, editUserPage }) => {
+    await page.goto('./dashboard');
+    await navbar.configurationButton.click();
+    await navbar.getNavItem('Users').click();
+    await userListPage.createUserButton.click();
+    await createUserPage.fillUserForm(formData);
+    await createUserPage.saveButton.click();
+    await expect(editUserPage.summary).toContainText(
+      `${formData.firstName} ${formData.lastName}`
+    );
+    await editUserPage.activateUser.click();
+    await editUserPage.saveButton.click();
+  }
+);
+
+test.afterEach(
+  async ({
+    page,
+    navbar,
+    loginPage,
+    locationChooser,
+    editUserPage,
+    userListPage,
+    mainLocation,
+  }) => {
+    await navbar.profileButton.click();
+    await navbar.logoutButton.click();
+    await loginPage.fillLoginForm(
+      AppConfig.instance.users.main.username,
+      AppConfig.instance.users.main.password
+    );
+    await loginPage.loginButton.click();
+    const location = await mainLocation.getLocation();
+    await locationChooser.getLocation(location.name).click();
```

Comment:
```suggestion
    await locationChooser.getOrganization(location.organization?.name).click();
    await locationChooser.getLocation(location.name).click();
```
We should also click on the organization tab otherwise we won't be able to click on the location.
It worked for you because the location that you provided in the env file has an organization that appears as a first tab which is open.
If you would have given another location as **main location** that has a different org. it would fail